pages/singlesite/[slug].js

```diff
@@ -210,8 +236,8 @@ const StyledSwiperSlide = styled(SwiperSlide)`
 const StyledImage = styled(Image)`
   display: block;
   object-fit: contain;
-  width: 80% !important;
-  height: 80% !important;
+  width: 100% !important;
```

Comment:
it seems that everything is very important ;) we usually try to avoid using `!important`.

Reply:
I am checking that !